CMakeLists.txt

```diff
@@ -47,12 +47,18 @@ message(STATUS "build type is ${CMAKE_BUILD_TYPE}")
 
 set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug;Release;RelWithDebInfo;MinSizeRel;Coverage")
 
-execute_process(
-	COMMAND git describe --always --tags
-	OUTPUT_VARIABLE git_tag
-	OUTPUT_STRIP_TRAILING_WHITESPACE
-	WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-	)
+if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/.git)
+        execute_process(
```

Comment:
This is super picky, but would you mind converting the spaces to tabs to be consistent with the rest of the file?

Reply:
Fixed. Also https://github.com/PX4/ecl/pull/852